controllers/Frontend.php

```diff
@@ -1,61 +1,71 @@
 <?php
 
-require_once "models/Article.php";
-require_once "models/Comment.php";
-require_once "models/Form.php";
-
 class Frontend
 {
     public function home_cv()
     {
-        $model_form = new Form();
+        require_once "config/formulaires.php";
+        $formulaire = form_page_home_cv();
         
-        $page="frontend/home_cv";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('home_cv', 'frontend');
+        $myView->render();
     }
 
     public function legalnotice()
     {
-        $page="frontend/legalnotice";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('legalnotice', 'frontend');
+        $myView->render();
     }
     
     public function home()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-
-        $page="frontend/home";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog();
+        
+        $myView = new View('home', 'frontend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function blog()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog_all();
         
-        $page="frontend/blog";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('blog', 'frontend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function post()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-                
-        $page="frontend/post";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        if(isset($_GET['id']))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)